features/host.yml

```diff
@@ -0,0 +1,10 @@
+name: Host
+description: The `::host` CSS pseudo-element selects the containing element of the shadow tree in which it is used.
```

Comment:
Unclear if the description should be extended to include `::host()` and `::host-context()` explicitly. I left them off for brevity

Reply:
I think `:host-context()` should be a separate feature, based  on support, and `:host()` should be mentioned in the description here.